packages/plugin-terraform/src/index.ts

```diff
@@ -74,10 +76,31 @@ export const plugin: PluginFunction<PluginConfig> = (
         hostname,
         visualization,
         schemaSuffix,
+        addAmplienceIsManagedSwitch: add_amplience_is_managed_switch,
       }),
     },
   });
 
+  // Add the managed switch to the terraform generator
+  if (add_amplience_is_managed_switch) {
+    for (const block of tfg.getBlocks()) {
+      let args = block.getArguments();
+      args = {
+        ...(add_amplience_is_managed_switch
+          ? { count: arg(`var.${amplienceIsManagedSwitchName} ? 1 : 0`) }
+          : {}),
+        ...args,
+      };
+      block.setArguments(args);
```

Comment:
Deze repo volgt zoveel mogelijk de principes van FP.
Immutability is daar eentje van.
Als data maar op 1 plek gezet wordt en nergens anders in de codebase wordt aangepast, is het makkelijker om overzicht te behouden en te redeneren over de data transformaties.

Gaarne deze data dus zetten in de transformatie functies van de visitor.